src/adapters/interfaces/IMorphoMarketV1Adapter.sol

```diff
@@ -38,27 +37,16 @@ interface IMorphoMarketV1AdapterBase is IAdapter {
     function parentVault() external view returns (address);
     function asset() external view returns (address);
     function morpho() external view returns (address);
-    function positions(Id id) external view returns (uint128 supplyShares, uint128 allocation);
+    function marketIds(uint256 index) external view returns (Id);
+    function positions(Id marketId) external view returns (uint128, uint128);
```

Comment:
If we are not going to return a `MarketPosition` struct, then we might as well move the definition of that struct to the implementation

Reply:
I see your point, but it's also about "where to go to get definitions". It's nice to know in advance you open `I<something>` to get the structs.

Reply:
2 getters vs returning `(uint128, uint128)` doesn't go one way or the other wrt where `MarketPosition` should be defined.

It's lower cost to do 1 call & throw away a value than to do 2 calls and not too ugly in code so I would keep the tuple.

Reply:
let's return the struct actually, it's not very costly

Reply:
#822